src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -491,7 +491,7 @@ int SetAdd_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
   if (changed) {
     return Set_DoPublish(ctx, argv, /*is_add=*/true);
   }
-  return REDISMODULE_OK;
+  return RedisModule_ReplyWithSimpleString(ctx, "OK");
```

Comment:
This fixes a bug in #4199: The `RAY.SET_ADD` command have no reply if trying to add a duplicate entry.

Reply:
I think either way would be ok.